src/Sql/Sql/Database Backup/Cmdlet/SetAzureSqlDatabaseBackupShortTermRetentionPolicy.cs

```diff
@@ -28,12 +29,17 @@ public class SetAzureRmSqlDatabaseBackupShortTermRetentionPolicy : AzureSqlDatab
         /// <summary>
         /// Gets or sets backup retention days.
         /// </summary>
-        [Parameter(Mandatory = true,
-            Position = 3,
```

Comment:
I believe the breaking changes about parameter sets are caused by the removal of `Position = 3`.
Changing a positional parameter to a normal parameter is a breaking change -- user may omit parameter name in their script, but now it's not allowed

Reply:
So it doesn't matter which position it is since parameter value always after parameter name. 
How can we skip these breaking changes in this PR? Looks like there is no real breaking changes.

Reply:
@isra-fel